securitas/templates/group.html

```diff
@@ -1,125 +1,101 @@
 
 {# Here it is relatively safe to assume that 'ipa' exists and is valid. We would not be here otherwise. #}
 
 {% block title %}Group: {{ group.name }}{% endblock %}
 
-{% block head %}
-  {{ super() }}
-  {{ css("group") }}
-{% endblock %}
-
 {% block content %}
   {{ super() }}
-
-  {% if current_user_is_sponsor and sponsor_form %}
-    {# Here we create the modal to add users to the group, but only if the person
-       viewing this page is a sponsor of the group. #}
-
-    <div id="sponsor-modal" class="modal with-typeahead">
-      <div class="modal-content">
-        <h4>Add Member to {{ group.name }}</h4>
-
-        <form action="{{ url_for('group_add_member', groupname=group.name) }}" method="post">
-          <div class="row">
-            <div class="input-field col s6">
-              {{ sponsor_form.hidden_tag() }}
-              {{ sponsor_form.new_member_username(class='typeahead', autocomplete="off") }}
-              {{ sponsor_form.new_member_username.label }}
-              <button class="btn waves-effect waves-light light-blue darken-4 w100" id="submit" type="submit" tabindex="9">Submit</button>
-            </div>
-          </div>
-        </form>
-      </div>
-    </div>
-  {% endif %}
-
   <div class="container section">
-
     <div class="row" data-section="identity">
-      <div class="col s12">
+      <div class="col">
         <h3 class="header light-blue-text text-darken-4 fw300">{{ group.name }}</h3>
         {% if group.description %}<p>{{ group.description }}</p>{% endif %}
       </div>
     </div>
-
     <div class="row">
-      <div class="col m6 s12" data-section="sponsors">
-        <h4 class="header fw300">Sponsors</h4>
+      <div class="col-12" data-section="sponsors">
+        <h4>Sponsors</h4>
         {% if sponsors|length == 0 %}
         <p>No sponsors yet.</p>
         {% else %}
-        <ul class="collection">
+        <div class="row">
           {% for sponsor in sponsors|sort(attribute='username') %}
-            <li class="collection-item avatar">
-              <!-- TODO: .red if sponsor/admin -->
-              {% if sponsor.mail %}
-                <img src="{{ gravatar(sponsor.mail, 200) }}" class="circle">
-              {% else %}
-                <i class="material-icons circle">person</i>
-              {% endif %}
-              <a href="{{ url_for('user', username=sponsor.username) }}">
-                <span class="title">{{ sponsor.username }}</span>
-              </a>
-              <p>{{ sponsor.name }}</p>
-              <p>sponsor</p>
-            </li>
+            <div class="col-4 mb-4">
+              <div class="card">
+              <div class="media align-items-center card-body">
+                {% if sponsor.mail %}
+                  <img src="{{ gravatar(sponsor.mail, 80) }}">
+                {% else %}
+                  <i class="fa fa-user fa-2x text-muted mr-3"></i>
+                {% endif %}
+                <div class="media-body">
```

Comment:
I think this could use some margin on the left, it seems to close to the avatar. Also, since the avatar is bigger this should be vertically centered I think.

Reply:
margin added. however, this one is already vertically centered, just the robot avatars make it look a little like they are not.

https://github.com/fedora-infra/securitas/pull/92/commits/63a03bf4e8a20b90fb08165c89114903ab14b8a1